tests/interactions/focusable-test.js

```diff
@@ -2,21 +2,28 @@
 import { expect } from 'chai';
 import { useFixture } from '../helpers';
 import { interactor, focusable } from '../../src';
+import { when } from '@bigtest/convergence';
 
 const FocusInteractor = interactor(function() {
   this.focusInput = focusable('.test-input');
 });
 
 describe('BigTest Interaction: focusable', () => {
-  let focused, test;
+  let focused, focusedIn, test;
 
   useFixture('input-fixture');
 
   beforeEach(() => {
     focused = false;
+    focusedIn = false;
 
-    document.querySelector('.test-input')
-      .addEventListener('focus', () => focused = true);
+    document.querySelector('.test-input').addEventListener('focus', e => {
```

Comment:
Last nit, let's make this a `$select` var so we don't have to repeat the `querySelector`